tests/Unit/Domain/CoordinateMaps/Test_Wedge3D.cpp

```diff
@@ -0,0 +1,265 @@
+        approx(-random_outer_radius_lower_eta / sqrt(3.0)));
+
+  CHECK(map_upper_zeta(inner_corner)[2] ==
+        approx(random_inner_radius_upper_zeta / sqrt(3.0)));
+  CHECK(map_lower_zeta(outer_corner)[2] ==
+        approx(-random_outer_radius_lower_zeta / sqrt(3.0)));
+
+  // Check that random points on the edges of the reference cube map to the
+  // correct edges of the wedge.
+  const std::array<double, 3> random_outer_face{
+      {real_dis(gen), real_dis(gen), 1.0}};
+  const std::array<double, 3> random_inner_face{
+      {real_dis(gen), real_dis(gen), -1.0}};
+  for (size_t i = 0; i < 3; ++i) {
+    CAPTURE_PRECISE(gsl::at(random_outer_face, i));
+    CAPTURE_PRECISE(gsl::at(random_inner_face, i));
+  }
+
+  CHECK(magnitude(map_upper_xi(random_outer_face)) ==
+        approx(random_outer_radius_upper_xi));
+  CHECK(map_upper_xi(random_inner_face)[0] ==
+        approx(random_inner_radius_upper_xi / sqrt(3.0)));
+  CHECK(magnitude(map_upper_eta(random_outer_face)) ==
+        approx(random_outer_radius_upper_eta));
+  CHECK(map_upper_eta(random_inner_face)[1] ==
+        approx(random_inner_radius_upper_eta / sqrt(3.0)));
+  CHECK(magnitude(map_lower_xi(random_outer_face)) ==
+        approx(random_outer_radius_lower_xi));
+  CHECK(map_lower_xi(random_inner_face)[0] ==
+        approx(-random_inner_radius_lower_xi / sqrt(3.0)));
+  CHECK(magnitude(map_lower_eta(random_outer_face)) ==
+        approx(random_outer_radius_lower_eta));
+  CHECK(map_lower_eta(random_inner_face)[1] ==
+        approx(-random_inner_radius_lower_eta / sqrt(3.0)));
+  CHECK(magnitude(map_lower_zeta(random_outer_face)) ==
+        approx(random_outer_radius_lower_zeta));
+  CHECK(magnitude(map_upper_zeta(random_outer_face)) ==
+        approx(random_outer_radius_upper_zeta));
+}
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Wedge3D.Jacobian",
+                  "[Domain][Unit]") {
+  // Set up random number generator
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_real_distribution<> real_dis(-1, 1);
+  std::uniform_real_distribution<> unit_dis(0, 1);
+  std::uniform_real_distribution<> inner_dis(1, 3);
+  std::uniform_real_distribution<> outer_dis(4, 7);
+
+  const double dxi = 1e-5;
+  const double deta = 1e-5;
+  const double dzeta = 1e-5;
+  const double xi = real_dis(gen);
+  CAPTURE_PRECISE(xi);
+  const double eta = real_dis(gen);
+  CAPTURE_PRECISE(eta);
+  const double zeta = real_dis(gen);
+  CAPTURE_PRECISE(zeta);
+  const double inner_radius = inner_dis(gen);
+  CAPTURE_PRECISE(inner_radius);
+  const double outer_radius = outer_dis(gen);
+  CAPTURE_PRECISE(outer_radius);
+  const double sphericity = unit_dis(gen);
+  CAPTURE_PRECISE(sphericity);
+
+  for (const auto direction : Direction<3>::all_directions()) {
+    const std::array<double, 3> delta_xi{{dxi, 0.0, 0.0}};
+    const std::array<double, 3> delta_eta{{0.0, deta, 0.0}};
+    const std::array<double, 3> delta_zeta{{0.0, 0.0, dzeta}};
```

Comment:
Unused

Reply:
Also add one test for a case where the sphericity is 1